solidity/test/maintainer/MaintainerProxy.test.ts

```diff
@@ -0,0 +1,3434 @@
+                        () => {
+                          let tx: ContractTransaction
+                          let initCallerBalance: BigNumber
+
+                          const targetWallets = liveWallets.slice(
+                            0,
+                            expectedTargetWalletsCount
+                          )
+
+                          before(async () => {
+                            await createSnapshot()
+
+                            initCallerBalance = await provider.getBalance(
+                              caller.address
+                            )
+
+                            tx = await maintainerProxy
+                              .connect(caller)
+                              .submitMovingFundsCommitment(
+                                ecdsaWalletTestData.pubKeyHash160,
+                                mainUtxo,
+                                walletMembersIDs,
+                                walletMemberIndex,
+                                targetWallets
+                              )
+                          })
+
+                          after(async () => {
+                            await restoreSnapshot()
+                          })
+
+                          it("should not revert", async () => {
+                            await expect(tx.wait()).not.to.be.reverted
+                          })
+
+                          it("should refund ETH", async () => {
+                            const postThirdPartyBalance =
+                              await provider.getBalance(caller.address)
+                            const diff =
+                              postThirdPartyBalance.sub(initCallerBalance)
+
+                            expect(diff).to.be.gt(0)
+                            expect(diff).to.be.lt(
+                              ethers.utils.parseUnits("1000000", "gwei") // 0,001 ETH
+                            )
+                          })
+                        }
+                      )
+                    }
+                  )
+                }
+              )
+            })
+          })
+        })
+      })
+    })
+  })
+
+  describe("resetMovingFundsTimeout", () => {
+    const walletDraft = {
+      ecdsaWalletID: ecdsaWalletTestData.walletID,
+      mainUtxoHash: ethers.constants.HashZero,
+      pendingRedemptionsValue: 0,
+      createdAt: 0,
+      movingFundsRequestedAt: 0,
+      closingStartedAt: 0,
+      pendingMovedFundsSweepRequestsCount: 0,
+      state: walletState.Unknown,
+      movingFundsTargetWalletsCommitmentHash: ethers.constants.HashZero,
+    }
+
+    context("when the wallet is in the MovingFunds state", () => {
+      before(async () => {
+        await createSnapshot()
+
+        await bridge.setWallet(ecdsaWalletTestData.pubKeyHash160, {
+          ...walletDraft,
+          state: walletState.MovingFunds,
+          movingFundsRequestedAt: (await lastBlockTime()) - 3600,
```

Comment:
Do we need to set it here? I am looking at the original test and this line is missing there.

Reply:
Simplified in https://github.com/keep-network/tbtc-v2/pull/209/commits/4c0b1457c3a24796e71f6d89165b96db3c097a09